qemu/tests/cfg/cpu_model_negative.cfg

```diff
@@ -0,0 +1,14 @@
+- cpu_model_negative:
+    type = cpu_model_negative
+    start_vm = no
+    warning_msg = ''
+    variants:
+        - enforce:
+            wrong_cmd = "-cpu 'Skylake-Client',+avx512_bf16,check,enforce"
```

Comment:
we need to find one available cpu model. Others look good to me.

Reply:
@nanliu-r Updated, thanks.